### DISCUSSION TITLE
Main Deployment Can't be Reached

### DISCUSSION THREAD

**Discussion by scherven** [vercel/community#4787] [Help]
### Summary

Hello,

I have an app that I am attempting to deploy to the following url: https://is-eurail-cheaper.vercel.app/. However, this URL does not work whatsoever, despite claiming to work just fine. 
<img width="476" alt="Screen Shot 2023-11-08 at 11 47 24 PM" src="https://github.com/vercel/community/assets/21976953/8a58c0e4-0394-45ed-b65f-660cb3150b05">
<img width="426" alt="Screen Shot 2023-11-08 at 11 47 36 PM" src="https://github.com/vercel/community/assets/21976953/45819e8c-692f-4941-af8c-e291f9637db9">
It was working a few days ago, and then just stopped for some reason. The other deployments work (ex: [is-eurail-cheaper-git-main-innoviox.vercel.app](https://is-eurail-cheaper-git-main-innoviox.vercel.app/)) but none of the functions work. Any help? Should I clear out some old deployments?

### Example

https://is-eurail-cheaper.vercel.app/

### Steps to Reproduce

1. Deploy app.
2. Go to production domain, fails to load.
3. Go to any other deployment, loads but functions to not run.

**Comment by swarnava** [comment]
Hi, What is the error are you getting when you trying to access the URL?

**Reply by scherven** [reply]
The URL simply doesn't load and eventually 404s.

**Reply by swarnava** [reply]
Hi, I just tried accessing https://is-eurail-cheaper.vercel.app/ and it worked successfully, can you send a screenshot?
